helm/cas-bciers/templates/coam/route.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{ template "cas-bciers.fullname" . }}-coam-frontend
+  labels:
+{{ include "cas-bciers.labels" . | indent 4 }}
+  annotations:
+    haproxy.router.openshift.io/balance: roundrobin
+    haproxy.router.openshift.io/rewrite-target: /
```

```suggestion
    haproxy.router.openshift.io/rewrite-target: {{ .Values.coamFrontend.route.path }}
```

You'll want to match the `rewrite-target` path with the host path to ensure NextJS routes the proper parts.